### ISSUE TITLE
Validation accuracy of pre-training on the tieredImageNet dataset

### ISSUE THREAD

**Issue by huangxilang901** [Tsingularity/FRN#7]
Dear authors

Thanks again for your great work and open source code.
Could you tell me the validation accuracy of  ResNet-12 backbone after pre-training on the tieredImageNet dataset?

Thank you

**Comment by Tsingularity** [comment]
Thanks for ur interest in our work!

On the original tiered-ImageNet dataset, the 5-shot validation accuracy is 86.01.
On the DeepEMD's tiered-ImageNet dataset, the 5-shot validation accuracy is 86.89.

Feel free to let me know if there's anything else we can help with.

**Comment by huangxilang901** [comment]
> Thanks for ur interest in our work!
> 
> On the original tiered-ImageNet dataset, the 5-shot validation accuracy is 86.01. On the DeepEMD's tiered-ImageNet dataset, the 5-shot validation accuracy is 86.89.
> 
> Feel free to let me know if there's anything else we can help with.

Dear author:

Sorry to bother you again.
Here's my question: When fine-tuning your method, did you fix the parameters of the pre-trained encoder?
I have looked into the codes but cannot find anything about frozening the parameters, so my opionion is that FRN directly applies the pre-trained encoder and adjusts the parameters to find the reconstruction matrix.
In some of the few-shot learning papers using pre-training, I noticed that some of them fixed the pre-trained encoder, while some of them did not.
Could you also share some comments about this? (sorry for this stupid question -_-)
Thank you in advance and hope you have a good day!

**Comment by Tsingularity** [comment]
That's a good question!

Your observations are correct. I also noticed that different papers use pre-trained backbones in different ways, sometimes fine-tuning with the same lr, sometimes with a smaller lr, sometimes even frozen. I think all these choices are performance-driven. I guess the authors might try them each and pick the one with the highest accuracy.

For FRN, during second-stage meta-training, we don't freeze the weights. We directly train it with a smaller lr and it usually is 1e-3. For different datasets/models, feel free to check the config files to see more hyper-param details.

Feel free to let me know if there's anything else we can help with.

**Comment by Tsingularity** [comment]
closing this for now. feel free to re-open it if you need more help.
